tests/benchmarks/README.md

```diff
@@ -43,7 +40,11 @@ It helps avoid common benchmarking pitfalls and provide highly-accurate, repeata
 
 A BDN benchmark is an executable. To run it, simply run `dotnet run %BenchmarkProject.fsproj%` in the benchmark's directory.
 
-### Writing a new benchmark
+## Quickly validating that the benchmarks work
+
+`SmokeTestBenchmarks.ps1` allows to run faster BDN benchmarks with a minimum number of iterations, as a way to verify that the benchmarks still work. This doesn't validate the notebook-based meta-benchmarks.
```

Comment:
There also needs to be a shell script, for people who don't develop on Windows.

Reply:
Done